pkg/cnab/provider/uninstall.go

```diff
@@ -1,76 +1,70 @@
 package cnabprovider
 
 import (
-	"fmt"
-
-	"get.porter.sh/porter/pkg/cnab/extensions"
-	"get.porter.sh/porter/pkg/manifest"
 	"github.com/cnabio/cnab-go/action"
+	"github.com/cnabio/cnab-go/bundle"
 	"github.com/cnabio/cnab-go/claim"
 	"github.com/pkg/errors"
 )
 
-func (d *Runtime) Uninstall(args ActionArguments) error {
-	c, err := d.claims.Read(args.Claim)
-	if err != nil {
-		// Yay! It's already gone
-		if err == claim.ErrClaimNotFound {
-			return nil
-		}
-		return errors.Wrapf(err, "could not load bundle instance %s", args.Claim)
-	}
+func (r *Runtime) Uninstall(args ActionArguments) error {
+	// TODO: (carolynvs) consolidate logic across actions
```

Comment:
Has this item been completed?  (It seems so.)

Reply:
I was thinking that I could consolidate the implementation of install/upgrade/invoke/uninstall into a single function, there's a lot of copy paste that is easy to mess up and not realize it. But there also aren't any unit tests either, just integration tests. Let me take a stab at it and see if I can change these safely.

Reply:
OK! Just pushed a commit with some unit tests and all the install/upgrade/invoke/uninstall consolidated.

Reply:
👏  https://github.com/deislabs/porter/pull/1065/commits/487403ae297af20a1edc8436536ad5abafe137c5 looks great!